client/protocols/unlock.py

```diff
@@ -71,17 +75,20 @@ def run(self, messages):
                     pass
                 print()
                 break
-        messages['unlock'] = analytics
+        messages['unlock'] = self.analytics
```

Comment:
Can we add a docstring as to what the analytics contains for this particular run method? It'll make it easier to make any changes in the future.